addons/web/static/src/core/emoji_picker/emoji_picker.xml

```diff
@@ -18,10 +18,11 @@
                 </span>
             </div>
             <t t-set="itemIndex" t-value="0"/>
-            <div class="o-EmojiPicker-content overflow-auto d-flex flex-grow-1 w-100 flex-wrap align-items-center user-select-none mt-1" t-att-class="getEmojis().length === 0 ? 'flex-column justify-content-center' : 'align-content-start'" t-ref="emoji-grid" t-on-scroll="highlightActiveCategory">
-                <t t-if="searchTerm and getEmojis().length === 0" class="d-flex flex-column">
+            <t t-set="allEmojis" t-value="getAllEmojis()"/>
```

Comment:
`allEmojis` seems to be a little bit generic and misleading.

Reply:
I renamed to `getEmojisFromSearch`/`emojisFromSearch`. Its value handles non-search, with trailing recent emojis  and they are duplicated from their presence in other sections, but in practice this is only used and relevant for search so better have a more flavored name while keeping code very simple with `[...recentEmojis, ...this.getEmojis()]`